### DISCUSSION TITLE
Anyway to have stash scan an external drive?

### DISCUSSION THREAD

**Discussion by shinobukid** [stashapp/stash#2147] [Q&A] [answered]
I Store my collection on a external Hard drive and i would like to know a way to have it scan that instead

**Comment by kermieisinthehouse** [comment, marked as answer]
You can add an extra Stash directory under Settings > Configuration > Stashes, and then rescan

**Reply by shinobukid** [reply]
I Dont see my external drive in the directory settings

**Reply by kermieisinthehouse** [reply]
You need to figure out the path to your external drive, then add that path as a new directory under "Stashes". I can be more specific, what platform are you on?

**Reply by shinobukid** [reply]
Im on windows

**Reply by kermieisinthehouse** [reply]
You'll need to fine the Drive letter, e.g. `F:\`, and then type that into the new directory popup. From there, you can navigate to the directory you want.

**Reply by shinobukid** [reply]
this worked thank you !
